### PR TITLE
Add shared renovate configuration

### PR DESCRIPTION
**Pull request by internalsystemerror**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | yes
| RFC           | no
| QA            | yes

### Description

This PR will allow easy onboarding for the Mezzio organisation, reusing the configuration from the Laminas organisation.

### CODE REVIEW

File: renovate-config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Default configuration for repositories in the Mezzio organisation",
+    "extends": [
+        "local>laminas/.github:renovate-config"
```
Comment (Ocramius) [inline renovate-config.json:5]: Is this indirection fine? Overall OK from me, just making sure we don't create too many hops for renovate to follow :D

**Review by Ocramius — APPROVED** [review]
Thanks @internalsystemerror!

File: renovate-config.json (hunk 1 above)
Comment (internalsystemerror) [reply renovate-config.json:5]: No no, it's just the 1 hop from laminas to here and to Laminas API Tools. It let's us share that config within these orgs.

**Comment by internalsystemerror** [comment]
@Ocramius Thanks for the review and merge!

**Pull request merged**